style.css

```diff
@@ -28,6 +32,33 @@ header {
   padding: 5px;
 }
 
+.mobile-menu {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  align-items: start;
+  padding: 15% 10%;
+  position: absolute;
+  z-index: 2;
+  width: 100vw;
+  height: 100vh;
+  left: 0;
+  top: 0;
+  background-color: #6070ff;
+  opacity: 0.9;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 44px;
+  gap: 32px;
+  backdrop-filter: blur(20px);
+}
+
+.mobile-menu a {
+  text-decoration: none;
+  color: #fff;
+}
```

Comment:
You did well with the implementation of this project, kindly implement the following changes so your design looks better and attractive 😄

Please I'd suggest you add the blurred background of the mobile popup, as you are expected to follow the design as closely as possible

Your design :
![image](https://github.com/Estete9/Fullstack-Portfolio/assets/20537712/fa94f517-dd7e-4200-b91e-d8299e130a41)

Figma design :
![image](https://github.com/Estete9/Fullstack-Portfolio/assets/20537712/9a80e813-1c94-46df-8468-08a5d6bc5085)